drivers/overlay/overlay.go

```diff
@@ -431,6 +439,25 @@ func (d *Driver) Get(id string, mountLabel string) (s string, err error) {
 		return "", err
 	}
 
+	newlowers := ""
+	for _, lower := range strings.Split(string(lowers), ":") {
+		newpath := path.Join(d.home, lower)
+		if _, err := os.Stat(newpath); err != nil {
+			for _, p := range d.AdditionalImageStores() {
+				lower := path.Join(p, d.name, lower)
+				_, err = os.Stat(lower)
+				if err == nil {
+					break
+				}
+			}
```

Comment:
If the loop doesn't exit via `break` at line 450, what value is expected to be appended to the `newlowers` list?